tests/test_draaitabel.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+from applicatie.logic.draaitabel import DraaiTabel
+from applicatie.logic.codelijst import Codelijst
+
+
```

Comment:
Nice dat je een unit test hebt gemaakt. Die hebben we inderdaad nodig.

Reply:
Thanks! :)